website/docs/r/group_member.md

```diff
@@ -41,13 +41,13 @@ The following arguments are supported:
 
 In addition to the above arguments, the following attributes are exported:
 
-* `etag`
+* `etag` - ETag of the resource.
 
-* `kind`
+* `kind` - Kind of resource this is.
 
-* `status`
+* `status` - Status of member.
 
-* `type`
+* `type`- Type of member.
```

Comment:
The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L2513